### PR TITLE
Bug 1625451 - Provide default for application field

### PR DESCRIPTION
**Pull request by octavian-negru**
[Back to Bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1625451)

### CODE REVIEW

**Comment by octavian-negru** [comment]
Local tests are passing, waiting for Travis to do its job...

File: treeherder/perf/models.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ class PerformanceSignature(models.Model):
     option_collection = models.ForeignKey(OptionCollection, on_delete=models.CASCADE)
     suite = models.CharField(max_length=80)
     test = models.CharField(max_length=80, blank=True)
-    application = models.CharField(max_length=10, null=True,
+    application = models.CharField(max_length=10, default='', blank=True,
```
Comment (ionutgoldan) [inline treeherder/perf/models.py:43]: The `application` field will only be provided via the background workers.
`blank=True` targets something entirely different; it's mostly concerned with forms originating from the frontend.

Comment (octavian-negru) [reply treeherder/perf/models.py]: Done.

**Comment by ionutgoldan** [comment]
@octavian-negru could you rebase on lastest master?

**Comment by ionutgoldan** [comment]
The equivalent raw SQL for the migration script looks like:
```sql
--
-- Alter field application on performancesignature
--
ALTER TABLE `performance_signature` ALTER COLUMN `application` SET DEFAULT '';
UPDATE `performance_signature` SET `application` = '' WHERE `application` IS NULL;
ALTER TABLE `performance_signature` MODIFY `application` varchar(10) NOT NULL;
ALTER TABLE `performance_signature` ALTER COLUMN `application` DROP DEFAULT;
```

**Comment by ionutgoldan** [comment]
Hi @sarah-clements! Could you take a look over this PR?
Thanks in advance!

**Pull request merged**